src/Tizen.NUI/src/public/WatchView.cs

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System.ComponentModel;
+
+namespace Tizen.NUI
+{
+    using System;
+    using System.Runtime.InteropServices;
+    using Tizen.NUI.BaseComponents;
+
+    /// <summary>
+    /// The WatchView is a class for displaying the watch.<br />
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class WatchView : View
+    {
+        /// <summary>
+        /// Creates a new WatchView.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public WatchView(Window window, string watchId, string contentInfo, int width, int height) : this(Interop.WatchView.WatchView_New(Window.getCPtr(window), watchId, contentInfo, width, height), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+
+        }
+
+        internal WatchView(global::System.IntPtr cPtr, bool cMemoryOwn) : base(Interop.WatchView.WatchView_SWIGUpcast(cPtr), cMemoryOwn)
+        {
+        }
+        internal WatchView(WatchView handle) : this(Interop.WatchView.new_WatchView__SWIG_1(WatchView.getCPtr(handle)), true)
+        {
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+        }
+
+        /// <summary>
+        /// Pauses a given Watch.
+        /// </summary>
+        /// <returns>True on success, false otherwise.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public bool PauseWatch()
+        {
+            bool ret = Interop.WatchView.WatchView_PauseWatch(swigCPtr);
+            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
```

Comment:
Please don't write the if-statement in the single line. I am pretty sure that is against used convention and will be reported by one of our tools.